Sources/ArcGISToolkit/Components/Offline/PreplannedMapModel.swift

```diff
@@ -14,7 +14,7 @@
 
 import ArcGIS
 import OSLog
-import SwiftUI
+import Combine
```

Comment:
When I comment out this import, it still compiles.

```suggestion
```

It seems the import of `OSLog` implicitly introduced `Foundation`. I'm not sure if we should import Foundation here as well.

Reply:
Oh! Good catch. `OSLog` should not be imported; it has been superseded by `os` (which does not export Foundation, by the way). So the imports for this file should be as follows:

```swift
import ArcGIS
import Combine
import Foundation

internal import os
```